Source/Core/Core/AchievementManager.cpp

```diff
@@ -290,6 +291,27 @@ void AchievementManager::LoadGameSync(const ResponseCallback& callback)
     callback(resolve_hash_response);
     return;
   }
+  u32 old_game_id;
+  {
+    std::lock_guard lg{m_lock};
+    old_game_id = m_game_id;
+  }
+  if (new_game_id == old_game_id)
+  {
+    INFO_LOG_FMT(ACHIEVEMENTS, "Alternate hash resolved for current game {}.", old_game_id);
+    callback(ResponseType::SUCCESS);
+    return;
+  }
+  else if (m_game_id != 0)
```

Comment:
Should be `new_game_id`.

Reply:
Actually, I think it should be old_game_id. new_game_id is checked a few lines above, and this code is supposed to trigger if the game is changing, but not if there was no game already running.